Packages.props

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.4' ">
+    <PackageReference Update="Microsoft.AspNetCore.WebUtilities" Version="1.1.2" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
+    <PackageReference Update="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
+  </ItemGroup>
+  <!-- <ItemGroup Condition="'$(IsTargetingNetFx)' == 'true'">
```

Comment:
Let's set Newtonsoft.Json to version 10.0.3 as the global default and add overrides if they are needed. I think we should try to move the 12.0.1 back to 10.0.3 but there may be some cases were we need to stick to 6.0.8 for the older target frameworks but that would be an override if needed.